tests/boardfarm_plugins/boardfarm_prplmesh/devices/prplmesh_prplwrt.py

```diff
@@ -83,10 +115,12 @@ def check_status(self) -> bool:
     def prplMesh_check_state(self) -> bool:
         """ Check prplMesh status. Return True if operational."""
         self.sendline("/etc/init.d/prplmesh status")
-        match = self.expect(
-                ["operational", "FAIL", pexpect.EOF, pexpect.TIMEOUT],
-                timeout=10)
-        if match == 0:
+        self.expect(
+                ["(?P<main_agent>OK) Main agent.+"
+                 "(?P<wlan0>OK) wlan0.+"
+                 "(?P<wlan2>OK) wlan2", pexpect.TIMEOUT],
+                timeout=5)
```

Comment:
Just remembered, that i have seen that `/etc/init.d/prplmesh status` actually sets appropriate shell exit code. Since functionality is the same, readability same (at least to me) don't see much sense changing.